packages/keystone/cipher.js

```diff
@@ -0,0 +1,157 @@
+const crypto = require('crypto')
+
+const { isEmpty } = require('lodash')
+
+// since data is converted in hex, ':' shouldn't be in it
+const SEPARATOR = ':'
+const SUPPORTED_MODES = ['cbc', 'ctr', 'cfb', 'ofb', 'ecb']
+const SUGGESTIONS = {
+    'ecb': 'WARNING! Don\'t use "ecb" in real application as it is insecure!',
+    'cfb': 'Please, consider using "ctr" or "cbc"',
+    'ofb': 'Please, consider using "ctr" or "cbc"',
+}
```

Comment:
please, add suggested - DEFAULT_ALGORITHM to use when someone decide to add a new encrypted field